data/json/mapgen/s_fabricstore.json

```
@@ -0,0 +1,91 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "s_fabricstore" ],
+    "//": "Frabic Outlet",
+    "weight": 100,
+    "object": {
+      "fill_ter": "t_floor",
+      "rows": [
+        "------------------------",
+        "------------------------",
+        "------------------------",
+        "..p--####:::++:::#......",
+        "..p--#JJ|        #......",
+        "..p--#  |        #......",
+        "..p--#+||        #......",
+        "..p--# [Q  ccc  C#......",
+        "..p--#  Q       C#......",
+        "..p--#  Q  CCC  C#......",
+        "..p--#  c        #......",
+        "..p--#  c        #......",
+        "..p--#+|||i|ccccc#......",
+        "..p--# [[[ |   s%#......",
+        "..p--#  s  |     #......",
+        "..p--;   s |+||||#......",
+        ".....0  [[   +___#......",
+        "....z#|+||i|||_St#......",
+        "....z#d |    #####......",
+        "....z#ds|LlLl#4zzz......",
+        "....z##0######z.........",
+        "........................",
+        "........................",
+        "........................"
+      ],
+      "palettes": [ "standard_building_palette", "fabric_store_pallete" ],
```

```suggestion
      "palettes": [ "standard_building_palette", "fabric_store_pallete" ],
```
There's no real benefit to defining your own palette for a single map especially when you've got unincluded terrains below, it just makes it more annoying to edit bc you have to cross reference the two you can leave it as is tho